cobra/test/test_flux_analysis.py

```diff
@@ -291,22 +291,46 @@ def construct_ll_test_model(cls):
         test_model.add_reactions([EX_A, DM_C, v1, v2, v3])
         return test_model
 
+    def test_legacy_loopless_benchmark(self, benchmark):
+        test_model = self.construct_ll_test_model()
+        benchmark(lambda: construct_loopless_model(test_model).optimize(
+            solver="cglpk"))
+
     def test_loopless_benchmark(self, benchmark):
         test_model = self.construct_ll_test_model()
-        benchmark(lambda: construct_loopless_model(test_model).optimize())
 
-    def test_loopless(self):
+        def _():
+            with test_model:
+                add_loopless(test_model)
+                test_model.optimize(solver="optlang-glpk")
```

Comment:
Yes! Absolutely. GLPK includes a MIP solver. It does not support indicator variables like cplex or gurobi so you have to declare them by hand.